packages/cli/src/commands/app/diff.ts

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Command } from 'commander';
+import {
+  diffTemplateFiles,
+  handlers,
+  handleAllFiles,
+  inquirerPromptFunc,
+  makeCheckPromptFunc,
+  yesPromptFunc,
+} from '../../lib/diff';
+import { version } from '../../lib/version';
+
+const fileHandlers = [
+  {
+    patterns: ['packages/app/package.json'],
+    handler: handlers.appPackageJson,
+  },
+  {
+    patterns: [/tsconfig\.json$/],
```

Comment:
in several of these matchers, you add an end anchor $ but nothing that anchors the start (a separator or ^). Probably not a big deal, but just noting

Reply:
Yeah it's intentional, to match tsconfigs anywhere for example